go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/go-cleanhttp v0.5.2
 	github.com/hashicorp/go-version v1.6.0
-	github.com/hashicorp/hcl-lang v0.0.0-20221014125844-7eceda07a779
+	github.com/hashicorp/hcl-lang v0.0.0-20221028130326-b8c8114d3c7e
```

Comment:
Although the latest revision doesn't impact `terraform-schema`, you may want to bump it to [b20d628](https://github.com/hashicorp/hcl-lang/commit/b20d6288c7f33d2cf567fdc7352c222e535eeec8) which is the latest.